### PR TITLE
Event listener

### PR DESCRIPTION
**Pull request by hellochar**
make the palette input event handler code a little easier to comprehend

### CODE REVIEW

**Comment by hellochar** [comment]
so jasmine failed because `pip install -r requirements.txt` timed out while downloading some of the required libraries. I don't know how to restart a build, but all tests are passing locally

File: editor/static/editor/js/palette.js [hunk 1]
```diff
@@ -36,21 +36,22 @@ function Palette(editor) {
 }
 
 /*
- * Convenience method to bind a given event handler function when this palette has selection, and unbind it when it loses selection.
+ * Convenience method to bind a set of event handlers when this palette has selection, and unbind it when it loses selection.
  *
  * PARAMETERS
```
Comment (jamesmuerle) [inline editor/static/editor/js/palette.js]: replace this with the standard javadoc syntax of
@param set an object with event names as keys and corresponding handlers as values
@param element any selector accepted by jQuery()

**Comment by jamesmuerle** [comment]
Just make the quick documentation change. That should be enough to repush and fix the build.

**Comment by hellochar** [comment]
@strayhobo did some research into documentation formats for javascript and it seems the standard is [JSDoc](http://en.wikipedia.org/wiki/JSDoc) so I'm gonna go with that; it looks inspired by javadoc anyways so we're pretty good there

**Pull request merged**